app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -95,7 +95,7 @@ $include-html-global-classes: $include-html-classes;
 // $font-weight-normal: normal;
 // $font-weight-bold: bold;
 
-// $white       : #FFFFFF;
+$white       : #F2F3F4;
```

Comment:
Color `#F2F3F4` should be written as `#f2f3f4`<br>Variable names should be followed immediately by a colon